zero/src/bin/leader/cli.rs

```diff
@@ -54,49 +54,54 @@ pub(crate) enum Command {
     /// Reads input from stdin and writes output to stdout.
     Stdio {
         /// The previous proof output.
-        #[arg(long, short = 'f', value_hint = ValueHint::FilePath)]
+        #[arg(short = 'f', long, env="ZERO_BIN_PREVIOUS_PROOF", value_hint = ValueHint::FilePath)]
         previous_proof: Option<PathBuf>,
     },
     /// Reads input from a node rpc and writes output to stdout.
     Rpc {
         // The node RPC URL.
-        #[arg(long, short = 'u', value_hint = ValueHint::Url)]
+        #[arg(short = 'u', long, env="ZERO_BIN_RPC_URL", value_hint = ValueHint::Url)]
         rpc_url: Url,
         // The node RPC type (jerigon / native).
-        #[arg(long, short = 't', default_value = "jerigon")]
+        #[arg(
+            short = 'T',
+            long,
+            env = "ZERO_BIN_RPC_TYPE",
+            default_value = "JERIGON"
```

Comment:
Why have you changed to uppercase?

Reply:
Not sure how that happened... I was using a keybinding to force text to upper, and it looks like I somehow applied that to `default_value`.